cmd/cli/kubectl-kyverno/main.go

```diff
@@ -28,6 +29,7 @@ func main() {
 		apply.Command(),
 		test.Command(),
 		jp.Command(),
+		oci.Command(),
```

Comment:
Wouldn't be better to not register the command at all if experimental features are not enabled ?

Reply:
yeah, it'd be better, excellent.